Update norwegian locale

flugger opened the pull request:
The "for" is not needed for the sentence to make sense in Norwegian, and this will give the developer the chance to omit the "for" if wanted. This is also more consistent compared to the other locales in this repository.

Oire commented:
@rexxars, @sigurdga, your thoughts, please?  
@flugger, even if guys approve your change, you need to edit only the `src/locale/nb.js` and `src/test/locale/nb.js` files to get it merged in. Thanks!

rexxars commented:
+1

sigurdga commented:
Related languages using "for" or similar: swedish (sv), icelandic (is), nynorsk (nn)
Related languages not using "for" or similar: danish (da), faroese (fo), dutch (nl)

I usualy find it is handy to have "for" as part of the long version, but I know I have disliked it before and needed to code around it.

+1 (after some thinking)

We should probably get this change into nynorsk (nn) as well.

flugger commented:
Hey, thanks for the feedback. I changed the files inside src/ now, including the test files. I also changed the nynorsk (nn) version accordingly.

For some reason I see the locale/nb.js file still has recorded a change, looks like a newline has been added. Tried to remove it twice, but no luck.

Oire commented:
@flugger, now everything looks good. However, I believe you need to undo the changes to `locale/nb.js` and rebase. Thanks!

ichernev commented:
Thank you! Will merge on next release & drop the newline change in `locale/nb.js`

ichernev commented:
Merged in 5df9e1828ab9b48bc44d837c44f8f5014f687e8c
